ui/webui/test/check-storage

```diff
@@ -731,13 +742,61 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         i.next()
 
         # verify review screen
-        disk = "vda"
-        r.check_disk(disk, "16.1 GB vda (0x1af4)")
+        r.check_disk(dev, "16.1 GB vda (0x1af4)")
 
-        r.check_disk_row(disk, 1, "vda1, 1.05 MB: biosboot")
-        r.check_disk_row(disk, 2, "vda2, 1.07 GB: format as ext4, /boot")
-        r.check_disk_row(disk, 3, "root, 15.0 GB: format as btrfs, /")
-        r.check_disk_row(disk, 4, "home, 15.0 GB: mount, /home")
+        r.check_disk_row(dev, 1, "vda1, 1.05 MB: biosboot")
+        r.check_disk_row(dev, 2, "vda2, 1.07 GB: format as ext4, /boot")
+        r.check_disk_row(dev, 3, "root, 15.0 GB: format as btrfs, /")
+        r.check_disk_row(dev, 4, "home, 15.0 GB: mount, /home")
+
+        i.back(previous_page=i.steps.CUSTOM_MOUNT_POINT)
+        i.back()
+
+        # Checks for nested btrfs subvolume
+        tmp_mount = "/tmp/btrfs-mount-test"
+        m.execute(f"""
+        mkdir -p {tmp_mount}
+        mount {disk}3 {tmp_mount}
+        btrfs subvolume create {tmp_mount}/home/Movies
+        btrfs subvolume create {tmp_mount}/home/Movies/Good_Movies
+        btrfs subvolume create {tmp_mount}/home/Movies/Bad_Movies
+        umount {tmp_mount}
+        rmdir {tmp_mount}
+        """)
+        s.rescan_disks()
+        self.select_mountpoint(i, s, [(dev, True)])
+
+        self.select_row_device(1, f"{dev}2")
+        self.select_row_device(2, "root")
+        self.add_mount()
+        self.select_row_device(3, "home")
+        self.select_row_mountpoint(3, "/home")
+        self.add_mount()
+        self.select_row_device(4, "home/Movies")
+        self.add_mount()
+        self.select_row_device(5, "home/Movies/Good_Movies")
+        self.select_row_mountpoint(5, "/mnt/Good_Movies")
```

Comment:
Just a small nitpick:It would make more sense to mount this under `/home/Movies/Good_Movies` instead of `/mnt`. But it doesn't matter for this test case where the subvolume is actually mounted so feel free to just ignore this.

Reply:
Fixed.